platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -257,6 +254,8 @@ private MerkleStateRoot(@NonNull final MerkleStateRoot from) {
                 setChild(childIndex, childToCopy.copy());
             }
         }
+
+        this.platformState = new PlatformStateAccessorSingleton(this);
```

Comment:
Is a new object required here? One of the children in the loop right above was this platform state singleton. A copy of it has been already created in the loop. My assumption would be that copy should be used as `this.platformState`

Reply:
No longer relevant.